src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -39,7 +26,7 @@ $bcs-task-left-offset: 10 * $bcs-task-grid;
     @include avatar-badge-icon;
 
     border-width: 2px;
-    right: $bcs-task-grid;
+    right: -4px;
```

Comment:
`-$bcs-task-grid` ?

Reply:
as far as I can tell the relationship between the grid and the avatar offset was incidental, it's actually based on the border width

Reply:
ah, it actually comes from that `border-width` value then 👍